Algorithms/mentoring/src/main/java/co/com/globant/java/mentoring/BasicAlgorithmExercises1.java

```diff
@@ -80,4 +80,8 @@ public void loopSolution(){
         System.out.println(response);
     }
 
```

Comment:
**Line 2** Remove wildcard import: this is not a good practise because the JVM import all clasess under the package. 
Fix the `loopSolution` Method: Apply the Single Responsibility Principle.
**Line 71 to 80** -> Avoid the use of + to concat string this is not a good practice. because multiples objects are created.
use final static attributes to use as a constants. L48 , L53
Avoid the ***=** use instead **++** it is more clear to read.

Reply:
may you explain to me more about L48 and L53?